src/components/webhooks/DeliveryInfoDialogContent.tsx

```diff
@@ -1,53 +1,33 @@
 import React from 'react';
-import { withStyles } from '@material-ui/core/styles';
 import DialogContent from '@material-ui/core/DialogContent/DialogContent';
 import Tabs from '@material-ui/core/Tabs';
 import Tab from '@material-ui/core/Tab';
 import ReactMarkdown from 'react-markdown';
 import { DeliveryInfoDialogLazyContentQueryResponse } from './__generated__/DeliveryInfoDialogLazyContentQuery.graphql';
 
-const styles = theme => ({});
-
 interface Props {
   delivery: DeliveryInfoDialogLazyContentQueryResponse['webhookDelivery'];
 }
 
-class DeliveryInfoDialogContent extends React.Component<Props> {
-  state = {
-    value: 0,
-  };
-
-  handleChange = (event, value) => {
-    this.setState(prevState => ({
-      ...prevState,
-      value: value,
-    }));
-  };
+export default (props: Props) => {
+  let [state, setState] = React.useState(0);
```

Comment:
We don't use hooks at the moment. Let's be consistent and not introduce them for now.